Solutions/Phosphorus/Data Connectors/PhosphorusDataConnector.json

```diff
@@ -0,0 +1,144 @@
+                            "metricName": "Total events received",
+                            "legend": "Phosphorus Audit events",
+                            "baseQuery": "Phosphorus_CL"
+                        }
+                    ],
+                    "dataTypes": [
+                        {
+                            "name": "Phosphorus_CL",
+                            "lastDataReceivedQuery": "Phosphorus_CL \n | summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+                        }
+                    ],
+                    "connectivityCriteria": [
+                        {
+                            "type": "SentinelKindsV2",
+                            "value": [
+                                "APIPolling"
+                            ]
+                        }
+                    ],
+                    "availability": {
+                        "status": 1,
+                        "isPreview": false
+                    },
+                    "permissions": {
+                        "resourceProvider": [
+                            {
+                                "provider": "Microsoft.OperationalInsights/workspaces",
+                                "permissionsDisplayText": "read and write permissions are required.",
+                                "providerDisplayName": "Workspace",
+                                "scope": "Workspace",
+                                "requiredPermissions": {
+                                    "action": true,
+                                    "write": true,
+                                    "read": true,
+                                    "delete": true
+                                }
+                            }
+                        ],
+                        "customs": [
+                            {
+                                "name": "REST API Credentials/permissions",
+                                "description": "**Phosphorus API Key** is required. Please make sure that the API Key associated with the User has the Manage Settings permissions enabled.\n\n Follow these instructions to enable Manage Settings permissions.\n 1. Log in to the Phosphorus Application\n 2. Go to 'Settings' -> 'Groups'\n 3. Select the Group the Integration user is a part of\n 4. Navigate to 'Product Actions' -> toggle on the 'Manage Settings' permission. "
+                            }
+                        ]
+                    },
+                    "instructionSteps": [
+                        {
+                            "description": "**STEP 1 - Configuration steps for the Phosphorus API**\n\n Follow these instructions to create a Phosphorus API  key.\n 1. Log into your Phosphorus instance\n 2. Navigate to Settings -> API \n 3. If the API key has not already been created, press the **Add button** to create the API key\n 4. The API key can now be copied and used during the Phosphorus Device connector configuration"
```

Comment:
Here in the Configuration section as well.

Reply:
Same comment as above
`The Phosphorus Application's Domain Name is unique for each customer. So its not possible to know beforehand the customer's domain name to add the hyperlink. But all of our customer are aware of the Phosphorus API Key and will be able to perform the required steps without further direction.`